api/src/main/resources/workbench.yaml

```diff
@@ -2842,9 +2858,46 @@ definitions:
       ACCESS_TO_CARE,
       EDUCATION_LEVEL,
       INCOME_LEVEL,
+      OTHER,
+      RACE_ASIAN,
+      RACE_AA,
+      RACE_HISPANIC,
+      RACE_AIAN,
+      RACE_MENA,
+      RACE_NHPI,
+      RACE_MORE_THAN_ONE,
+      AGE_CHILDREN,
+      AGE_ADOLESCENTS,
+      AGE_OLDER,
+      AGE_OLDER_MORE_THAN_75
+    ]
+
+  disseminateResearchEnum:
+    type: string
+    description: disseminate findings
+    enum: [
+      PUBLICATION_PEER_REVIEWED_JOURNALS,
+      PRESENATATION_SCIENTIFIC_CONFERENCES,
+      PRESS_RELEASE,
+      PUBLICATION_COMMUNITY_BASED_BLOG,
+      PUBLICATION_PERSONAL_BLOG,
+      SOCIAL_MEDIA,
+      PRESENTATION_ADVISORY_GROUPS,
       OTHER
     ]
 
+  ResearchOutcomeEnum:
+    type: string
+    description: Research Outcomes
```

Comment:
[nit] Same here – leave blank, or expand this into something providing more context than the enum itself provides.